nautobot/core/templates/generic/object_bulk_create.html

```diff
@@ -25,7 +26,7 @@
 {% block tabs %}{% endblock %}
     <div class="row">
         <div class="col-md-8 col-md-offset-2">
-            <h1>{% block title %}{{ obj_type|bettertitle }} Bulk Import{% endblock %}</h1>
+            <h1><img src="{% static 'img/nautobot_chevron_header.svg' %}"/>{% block title %}{{ obj_type|bettertitle }} Bulk Import{% endblock %}</h1>
```

Comment:
As mentioned in standup - try the following css as an alternative to updating a bunch of templates:

```css
h1::before {
    content: url("/static/img/nautobot_chevron_header.svg");
    position: relative;
    top: 15px;
}
```

Note that for "correctness" we'd need a templated CSS file somewhere (since we shouldn't be hard-coding `/static/`) as a way to inject this (perhaps https://stackoverflow.com/a/55225412?) but it's a thought...!